Tests/MapboxMapsTests/Gestures/GestureHandlers/PinchGestureHandlerTests.swift

```diff
@@ -143,6 +143,69 @@ final class PinchGestureHandlerTests: XCTestCase {
         XCTAssertNil(mapboxMap.setCameraStub.parameters[2].bearing)
     }
 
+    func testPinchChangedWithZoomEnabledFalse() {
+        pinchGestureHandler.zoomEnabled = false
+
+        mapboxMap.cameraState = .random()
+
+        // these touches are consistent with the initial pinch midpoint
+        // and yield an angle of 45° for the initial state
+        gestureRecognizer.getNumberOfTouchesStub.defaultReturnValue = 2
+        gestureRecognizer.getStateStub.defaultReturnValue = .began
+        mapboxMap.dragCameraOptionsStub.defaultReturnValue = CameraOptions(
+            center: .random(),
+            zoom: .random(in: 0...20))
```

Comment:
drag camera options won't be invoked when handling the `.began` event. This can be removed.